docs/multicluster/user-guide.md

```diff
@@ -469,6 +469,16 @@ exported a Service: `default/nginx` with TCP Port `80`, other clusters can only
 export the same Service with the same Ports definition including Port names. At
 the moment, Antrea Multi-cluster supports only IPv4 multi-cluster Services.
 
+By default, the endpoints of multi-cluster Service will be exported Service's
+`ClusterIP`. Since Antrea v1.9.0, Antrea multi-cluster supports changing the type
+of multi-cluster Service's Endpoint. You can change the value of config
+`endpointIPType` in ConfigMap `antrea-mc-controller-config-***` from `ClusterIP`
+to `PodIP`. All member clusters should use the same type in a ClusterSet. Existing
+ServiceExports should be re-exported after changing `endpointIPType`. `ClusterIP`
+type requires that Multi-cluster Gateway is configured. `PodIP` type requires
```

Comment:
We should modify the `Preparation` section too.

> Since Antrea 1.7.0, multi-cluster Services require an Antrea Multi-cluster Gateway to be set up in each member cluster to route Service traffic across clusters, and Service CIDRs (ClusterIP ranges) must not overlap among clusters.

Since Antrea 1.7.0, multi-cluster Services require an Antrea Multi-cluster Gateway to be set up in each member cluster to route Service traffic across clusters, when using the `ClusterIP` endpoint type, or when using the `PodIP` type and there is no direct Pod-to-Pod connectivity across clusters (please refer to the [Multi-cluster Service section](#multi-cluster-service) for more information).

Reply:
Could you also add a sentence to the first paragraph of the quick-start.md about: We use the default `ClusterIP` endpoint type for multi-cluster Services in this guide.

And in its `Preparation` section:
> Since Antrea 1.7.0, multi-cluster Services require an Antrea Multi-cluster Gateway to be set up in each member cluster to route Service traffic across clusters, and Service CIDRs (ClusterIP ranges) must not overlap among clusters.

As we use `ClusterIP` endpoint type for multi-cluster Services, an Antrea Multi-cluster Gateway needs be set up in each member cluster to route Service traffic across clusters, and Service CIDRs (ClusterIP ranges) must not overlap among clusters.

Reply:
Done